tests/apps/auth/test_authentication.py

```diff
@@ -0,0 +1,70 @@
+from collections import namedtuple
+
+from unittest.mock import patch
```

Comment:
Did you install `pre-commit`? I think that this blank line between these imports it would be alerted